cmd/geth/flags.go

```diff
@@ -150,14 +152,36 @@ var (
 	}
 	LogDirFlag = DirectoryFlag{
 		Name:  "log-dir,logdir",
-		Usage: "Directory in which to write log files, redirecting terminal out (stderr)",
+		Usage: "Directory in which to write log files.",
 		Value: DirectoryString{filepath.Join(common.DefaultDataDir(), "logs")},
 	}
 	LogStatusFlag = cli.StringFlag{
-		Name: "log-status",
+		Name:  "log-status",
```

Comment:
Whitespace typo? Or is this intentional to only have two whitespace before `Name:`?

Reply:
Hmm okay I get it.